core/workspace_svg.js

```diff
@@ -1352,6 +1417,11 @@ Blockly.WorkspaceSvg.prototype.showContextMenu_ = function(e) {
         topBlocks));
   }
 
+  // Option to add a workspace comment.
+  if (this.options.comments) {
```

Comment:
👍

Reply:
Note that this is enabled on scratch blocks but not blockly.